src/plot_utils.py

```diff
@@ -633,7 +657,7 @@ def zmatrix(data_df, clustermap_kws=None, row_ordering=None,
         df = df.ix[:, col_ordering]
         df = df.ix[row_ordering, :]
         del clustermap_kws['pivot_kws']
-        fig, ax = plt.subplots()
+        _fig, ax = plt.subplots()
```

Comment:
And I'm trying to remind you that that horse is long ago out of the gate:
grep -r 'plt.' /Users/gremio/GoogleDrive/workspace/py27/lib/python2.7/site-packages/seaborn/ | wc
501

No sense in us worrying about it.